README.markdown

```diff
@@ -67,18 +67,18 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
     </tr>
 </table>
 
-Internet connectivity is required for downloading Terasology via the Launcher, afterwards offline play is possible.
+Internet connectivity is required for downloading Terasology via the Launcher, afterwards playing offline is possible.
 
 For easy game setup (recommended) you can use our launcher - [download it here.](https://github.com/MovingBlocks/TerasologyLauncher/releases)
 
 For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules.](docs/Modules.md)
 
 
-#### Alternate Install Method
+#### Alternative Installation Methods
 
-If you know you already have a Java Runtime Environment installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
+If you know you already have a Java Runtime Environment(JRE) installed, you may use a direct download release as an alternative for using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
 
-Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge develop version can be downloaded direct [here from our Jenkins](http://jenkins.terasology.org/job/DistroOmega/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip)
+Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge developed version can be downloaded direct [here from our Jenkins](http://jenkins.terasology.org/job/DistroOmega/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip)
```

Comment:
This one is a bit different. With "develop version" we try to refer to the `develop` branch. Maybe "development version" is a somewhat common term used for these kinds of builds?